features/step_definitions/role_steps.rb

```diff
@@ -82,7 +82,13 @@
   click_on using_design_system? ? "Edit#{role.name}" : role.name
   click_on "New appointment"
   select person_name, from: "Person"
-  select_date 1.day.ago.to_s, from: "Started at"
+  if using_design_system?
+    select 1.day.ago.day.to_s, from: "Day"
```

Comment:
there's a helper method you can use here 

```
  def fill_in_date_and_time_field(date)
    date = Time.zone.parse(date)

    select date.year, from: "Year"
    select date.strftime("%B"), from: "Month"
    select date.day, from: "Day"
    select date.strftime("%H"), from: "Hour"
    select date.strftime("%M"), from: "Minute"
  end
```

so all you'd need to input is 
```
within "#role_appointment_started_at" do
      fill_in_date_and_time_field(1.day.ago.to_s)
    end
```

Reply:
Cool, thanks @davidgisbey. I've changed fill_in_date_and_time_field to fill_in_date_fields.